src/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,103 @@
+  - Cloud
+---
+
+The Cloud infrastructure scales according to your resource needs to achieve greater efficiency. The {{site.data.var.ece}} monitors your applications and can adjust capacity to maintain steady, predictable performance. Converting to this architecture helps to mitigate problems, such as latency or large spikes in traffic.
+
+## Split architecture
+
+Historically, the Pro architecture consisted of 3 nodes, each containing a full tech stack. Now, there is a  scalable infrastructure that provides a tiered solution with a minimum of 6 nodes: 3 nodes for the web server and 3 nodes for the core database and other services. This split architecture provides the capability to scale tiers independently to achieve an optimal balance of performance. The **core** tier scales vertically (increases in size), and the **web** tier scales horizontally (increases instance count) and vertically (changes instance type and size).
+
+Scaling must use the same instance type and size for each node:
+
+-  `C5 instances` for each WEB node
+-  `M5 or M5n instances` for each Core node
+
+### Core tier scaling
+
+There are 3 nodes (core nodes) for the database and included services, such as ElasticSearch, MariaDB, and Redis. When the core tier approaches capacity, the only way to scale is to increase the server size, such as boosting the CPU power and memory. Capacity is limited to the size of the node that is available. Because the database cluster is designed for high availability, you cannot scale horizontally in a reliable way with the technologies used.
+
+![Core tier scaling]
+
+Consider an example that the core node instance type is _m5.2xlarge_ with 32Gb RAM. A service, such as the database, uses a considerable amount of memory (30Gb). Scaling to the next available instance size _m5.4xlarge_ provides 64Gb RAM, double the memory, and accommodates the growing needs of the database.
+
+Further optimizing performance includes routing traffic. You can route traffic based on the node type. By default the db node is isolated from the web traffic, so, as an example, you can choose to serve web traffic on the db node.
+
+### Web tier scaling
+
+There are 3 nodes for php-fpm and manages web traffic. In addition to vertical scaling by increasing power and memory, the web tier can scale horizontally by adding extra web servers to an existing cluster when constricted at the PHP level.
+
+![Web tier scaling]
+
+This complements the vertical scaling provided by the database tier.
+
+Consider an example that the web node instance type is _C5.2xlarge with 8 CPU and 16Gb RAM_. The number of requests to the site increased greatly. You can add an additional C5.2xlarge node to handle the increase in php-fpm processes or you can change each instance type to _C5.4xlarge with 16 CPU and 32Gb RAM_. Adding an additional node reduces the risk of insufficient surge capacity.
+
+## Project structure
+
+Minimally, Pro projects with the Scaling architecture have 6 nodes available.
```

Comment:
with the Scaling architecture --     title = Scaled architecture;  reference to split architecture, and scalable infrastructure...   Just wondering what the official term that customers will search on--   I think it might be helpful to clarify the relationship between split architecture and scaling --    Does split architecture = scaled architecture, or split architecture enables scaling, or a project with split architecture that has been scaled will have a scaled architecture?